lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,154 @@
+                                    model: accountSettingsModel,
+                                    title: gettext("Educational Background"),
+                                    valueAttribute: "level_of_education",
+                                    options: fields_data['level_of_education']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Gender"),
+                                    valueAttribute: "gender",
+                                    options: fields_data['gender']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Year of Birth"),
+                                    valueAttribute: "year_of_birth",
+                                    options: fields_data['year_of_birth']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Country"),
+                                    valueAttribute: "country",
+                                    options: fields_data['country']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Preferred Language"),
+                                    valueAttribute: "language",
+                                    helpMessage: gettext("If your preferred language isn't available as an edX language, you can indicate your preferance here."),
+                                    options: fields_data['preferred_language']['options'],
+                                })
+                            },
+                        ]
+                    },
+                    {
+                        title: gettext("Connected Accounts"),
+                        fields: [
+                            {
+                                view: new AccountSettingsFieldViews.LinkFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Facebook"),
+                                    valueAttribute: 'auth-facebook',
+                                    linkTitle: gettext("Link"),
```

Comment:
To  me, it would make more sense to have the words "Facebook" and "Google" be the links, rather than having a separate column with the word "Link". Thoughts?